.buildkite/lib/parseTemplate.ts

```diff
@@ -72,16 +72,16 @@ const validateFrontmatter = (meta: any): Frontmatter & { errors: string[] } => {
     errors.push("missing platforms");
   }
 
-  if (!meta.use_cases) {
-    errors.push("missing use_cases");
+  if (!meta.use_cases || meta.use_cases.length == 0) {
+    errors.push("must have at least one `use_cases`");
   }
 
   if (!meta.languages) {
```

Comment:
Out-of-scope for this PR, but should we be applying the same conditional to ensure all Frontmatter categories are arrays of strings?